include/snitch/snitch_config.hpp.config

```diff
@@ -89,6 +89,9 @@
 #if !defined(SNITCH_SHARED_LIBRARY)
 #cmakedefine01 SNITCH_SHARED_LIBRARY
 #endif
+#if !defined(SNITCH_DISABLE)
+#cmakedefine01 SNITCH_DISABLE
+#endif
```

Comment:
I think we should name this `SNITCH_ENABLE` rather than `SNITCH_DISABLE` (and invert the logic everywhere). Several reasons: 
 - avoid double-negatives in `!SNITCH_DISABLE`, 
 - all other on/off configuration variables are defined as "enable this" / "with that".

Reply:
Makes sense

Reply:
Done, see updates